spec/fixtures/model/force/lease/lease_domain.json

```diff
@@ -0,0 +1,12 @@
+{
+  "application_id": "a0o0P00000FFc2yQAD",
+  "id": "a130U0000001n4wQAA",
+  "leaseStartDate": "2028-01-01",
```

Comment:
I thought domain was snake_case? I believe other domain fixtures are.

Reply:
yeah, for some reason all the already-existing lease domain fields were in camel case when i came to this story. i think you and @chrisdolendo did the previous lease work - does anyone recall why this is? i'm fine with changing them to snake case in this PR if there's no problem with doing that. even though we may ultimately decide that we want to convert domain to camel case down the line, until then it would be better to maintain consistency

Reply:
yeah probably my fault . If you're up for it I think we should make it consistent. If you'd prefer, I can followup in a new PR after this one

Reply:
i'll do it in this PR, i got most of it done already during retro and sprint planning 😀

Reply:
done!